ui/admin/app/routes/scopes/scope/workers.js

```diff
@@ -37,4 +43,40 @@ export default class ScopesScopeWorkersRoute extends Route {
   refreshWorkers() {
     return super.refresh(...arguments);
   }
+
+  /**
+   * Save an worker in current scope.
+   * @param {UserModel} worker
```

Comment:
@param {WorkerModel} worker
```suggestion
   * @param {WorkerModel} worker
```

Reply:
wow thanks @DhariniJeeva